hello/src/clixon-hello@2019-04-17.yang

```diff
@@ -2,7 +2,7 @@ module clixon-hello {
     yang-version 1.1;
     namespace "urn:example:hello";
     prefix he;
-    revision 2019-04-17 {
```

Comment:
According to https://datatracker.ietf.org/doc/html/rfc7950#section-7.1.9 new revisions should be added before existing.